plugin/build.gradle

```diff
@@ -25,12 +63,6 @@ android {
             abiFilters "arm64-v8a"
         }
     }
-    externalNativeBuild {
-        cmake {
-            path file('CMakeLists.txt')
```

Comment:
We still need this so Android Studio knows which CMakeLists.txt file to use as reference.
You want to disable the build tasks tied to it though; for reference see https://github.com/godotengine/godot/blob/master/platform/android/java/nativeSrcsConfigs/build.gradle#L45-L56